libcob/ChangeLog

```diff
@@ -1,4 +1,11 @@
 
+2025-09-09  David Declerck <david.declerck@ocamlpro.com>
+
+	* intrinsic.c (cob_intr_char, cob_intr_ord): consider
+	  the program collating sequence in CHAR and ORD
+	* intrinsic.c (cob_intr_char): raise COB_EC_ARGUMENT_FUNCTION
+	  when calling CHAR with an argulent outside of the 1-256 range
```

Comment:
typo: argument
maybe just say "outside of the collation range" (as noted, we may use a new cob_intr_col in this case - and could even do this depending on the setup of a program collation, which removes the checks on this var as well)